### PR TITLE
Add guide in readme

### PR DESCRIPTION
**Pull request by deku-95**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,71 @@
...
+Optionally, you can add a [`ros test`](https://github.com/ros-tooling/setup-ros) job before the artefacts-ci to ensure everything is working before running the platform job:
+    
+```
+jobs:
+  ros2-test:
+    runs-on: ubuntu-latest
+    env:
+      ROS_DISTRO: humble
+    steps:
+        - uses: ros-tooling/setup-ros@v0.6
```
Comment (fabid) [inline README.md:30]: I guess a checkout would be needed here as well?

**Review by fabid — APPROVED** [review]
Thank you, great improvement. 1 comment related to the advanced usage, otherwise good to merge

File: README.md (hunk 1 above)
Comment (fabid) [reply README.md:30]: not sure about dolly, but here, either we add the checkout, or we skip this section of the README

Comment (fabid) [reply README.md:30]: @deku-95 Can you add the checkout step so that we can merge the PR

Comment (deku-95) [reply README.md:31]: Done

**Review by fabid — APPROVED** [review]

**Pull request merged**